lib/simplifyExpression/collectAndCombineSearch/LikeTermCollector.js

```diff
@@ -220,6 +223,19 @@ function getTermsForCollectingMultiplication(node) {
 }
 
 // A helper function for getTermsForCollectingMultiplication
+// e.g. nthRoot(x, 2), append '2': nthRootNode to terms dictionary
```

Comment:
@aliang8 can you make this change? It actually feels important to me. Having just '2' as a key in a dictionary feels like a bad idea. Also it kind of implies that '2' is the common term that we're grouping under, but it's not. I agree with Evy's suggestion above about the constant :)